test/DropdownButtonSpec.js

```diff
@@ -486,19 +517,27 @@ describe('DropdownButton', function() {
       buttonNode.getAttribute('aria-expanded').should.equal('true');
     });
 
-    it('when open and the key "esc" is pressed the menu is closed and focus is returned to the button', function() {
+    // I am not sure why this test has to be so convoluted.
+    // the naive approach would fail _only_ if the entire test suite was run together, but I couldn't isolate it any more
```

Comment:
Ok, that makes sense. Could you perhaps take a moment and re-word that comment to reflect that more clearly?